src/app/[...recipient]/client.tsx

```
@@ -0,0 +1,242 @@
+                message: 'You can pay the recipient in their preferred token',
+                buttonText: 'Pay them in their preferred token',
+                redirectTo: `/${error.recipient}`,
+            }
+        case EParseUrlError.INVALID_AMOUNT:
+            return {
+                title: 'Invalid Amount',
+                message: 'Please check the url and try again',
+                buttonText: isUser ? 'Go to home' : 'Create your Peanut Wallet',
+                redirectTo: isUser ? '/home' : '/setup',
+            }
+        case EParseUrlError.INVALID_URL_FORMAT:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)